code/modules/jobs/job_types/clown.dm

```diff
@@ -30,14 +30,18 @@
 
 	minimal_lightup_areas = list(/area/crew_quarters/theatre)
 
+/datum/job/clown/New()
+	. = ..()
+	if(HAS_TRAIT(SSstation, STATION_TRAIT_CLOWN_BRIDGE))
+		outfit = /datum/outfit/job/clown/bridge
+
 /datum/job/clown/after_spawn(mob/living/carbon/human/H, mob/M, latejoin = FALSE, client/preference_source, on_dummy = FALSE)
 	. = ..()
-	if(!ishuman(H))
-		return
-	if(!M.client || on_dummy)
+	if(!ishuman(H) || !M.client || on_dummy)
 		return
 	H.apply_pref_name(/datum/preference/name/clown, preference_source)
-
+	if(HAS_TRAIT(SSstation, STATION_TRAIT_CLOWN_BRIDGE))
+		to_chat(M, "<span class='notice bold'>The <span class='honk bold'>Clown Planet</span> has given all clowns access to a specific weakness in airlock ID scanners, resulting in all clowns having <b>bridge access</b>! Honk!</span>")
```

Comment:
This should just be on the bridge clown outfit instead, since this clown outfit doesn't respect STATION_TRAIT_CLOWN_BRIDGE

Reply:
The outfit is equipped before the player's client is put into the character.

Reply:
I think he meant the announcement should be in after outfit equipment.

Reply:
What do you mean?

Reply:
Moving it to `outfit.after_equip()`(something like that) proc from `job.after_spawn()` there.

Reply:
The issue is that I'm pretty sure `after_equip` would run _before_ the ckey gets put into the character.